app/src/androidTest/java/com/android/sample/screen/WorkoutCreationScreenTest.kt

```diff
@@ -127,15 +120,14 @@ class WorkoutCreationScreenTest {
     composeTestRule
         .onNodeWithTag("selectedExerciseType")
         .assertTextEquals("Selected Exercise: ${ExerciseType.PUSH_UPS}")
-    composeTestRule.onNodeWithTag("timeBasedButton").assertIsDisplayed().assertHasClickAction()
-    composeTestRule
-        .onNodeWithTag("repetitionBasedButton")
-        .assertIsDisplayed()
-        .assertHasClickAction()
-    composeTestRule.onNodeWithTag("timeBasedButton").performClick()
-    composeTestRule.onNodeWithTag("durationTextField").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("setsTextField").assertIsDisplayed()
-    composeTestRule.onNodeWithTag("repetitionBasedButton").performClick()
+
+    // check that PushUps is a repetition based exercise
+
+    // composeTestRule.onNodeWithTag("durationTextField").assertIsDisplayed()
+    // composeTestRule.onNodeWithTag("setsTextField").assertIsDisplayed()
+    // composeTestRule.onNodeWithTag("repetitionBasedButton").performClick()
+    composeTestRule.onNodeWithTag("repetitionsTextField").assertIsDisplayed()
+    // check default value
```

Comment:
You should consider removing the commented tests if they're not checked anymore. Also the assertion for "repetitionsTextField" is present twice.